test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -132,6 +132,93 @@ Describe "Format-Table" -Tags "CI" {
                 $result | Should -Match "Jim\s+5678\s+False"
         }
 
+        It "Format-Table output should not be greater than console width" -Pending: ([console]::WindowWidth -eq 0) {
+            $consoleWidth = [console]::WindowWidth
```

Comment:
This can often be 0 in Test Automation

Reply:
You can get how PowerShell thinks the UI is:
`$host.UI.RawUI.WindowSize.Width`

PowerShell itself deals with the `0` width console.